docs/api/main-config-tags.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'tags'
+hideRendererSelector: true
+---
+
+Parent: [main.js|ts configuration](./main-config.md)
+
+Type: `{ [key: string]: { description?: string } }`
+
+[Tags](../writing-stories/tags.md) allow you to control which stories are included in your Storybook. They are defined in the `tags` property of the Storybook configuration. Then they can be [applied to stories](../writing-stories/tags.md#tagging-stories) at the project, component, and story levels using the `tags` array.
+
+## Default tags
+
+The following tags are available by default in your project:
+
+```js
+{
+  dev: { description: 'Rendered in the Storybook UI, but only in development mode. They do not appear in the sidebar in production mode.' },
+  docs: { description: 'Rendered in the Storybook UI, but only in the docs page. They do not appear in the sidebar.' },
+  test: { description: 'Not rendered in the Storybook UI, in either development or production mode.' },
+}
```

Comment:
Is there more involved to each definition? e.g. I've seen reference to a `hidden` property.